org.videolan.VLC.json

```diff
@@ -700,13 +723,13 @@
       ]
     },
     {
-      "name": "libmodplug",
+      "name": "nasm",
```

Comment:
`nasm` is in the runtime.

Reply:
The one there is too old for recent x264.